src/boot/boot.c

```diff
@@ -54,6 +54,12 @@ typedef enum LoaderType {
         _LOADER_TYPE_MAX,
 } LoaderType;
 
+typedef enum {
+        REBOOT_NO = false,
+        REBOOT_YES = true,
+        REBOOT_AUTO,
+} reboot_on_err;
```

Comment:
we usually use CamelCase for type names. and we name the enum and the typedef the same usually. also, we don't like needless abbreviations. Hence change this to RebootOnError please